src/components/UsersTableItem/UserTableItem.spec.tsx

```diff
@@ -13,11 +13,9 @@ const link = new StaticMockLink(MOCKS, true);
 const link2 = new StaticMockLink(MOCKS2, true);
 const link3 = new StaticMockLink(MOCKS_UPDATE, true);
 import useLocalStorage from 'utils/useLocalstorage';
-import {
-  REMOVE_ADMIN_MUTATION,
```

Comment:
Why was this removed?

Reply:
declared but not used in the code.
<img width="865" alt="image" src="https://github.com/user-attachments/assets/cfee6bac-4690-400e-ab8a-b773c7431c5e" />